src/containers/Transactions/Transactions.container.tsx

```
@@ -0,0 +1,124 @@
+    columnId: string,
+    theme: Theme,
+): JSX.Element => {
+    switch (columnId) {
+        case 'transaction':
+            const paymentText =
+                'Payment ' +
+                (row.type === 'pay'
+                    ? 'from '
+                    : row.type === 'refund'
+                      ? 'refund to '
+                      : 'failed from ');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)